README.md

```diff
@@ -19,217 +19,172 @@
+Gadgets are built using the [`ig image build`](/docs/gadg) command.
+
+You can find a growing collection of Gadgets on [Artifact HUB](https://artifacthub.io/packages/search?kind=22). This includes both in-tree Gadgets (hosted in this git repository in the [/gadgets](/gadgets/README.md) directory and third-party Gadgets).
+
+See the [Gadget documentation](/gadgets/README.md) for more information.
+
+#### :warning: For versions prior to v0.31.0
+
+Prior to v0.31.0, Inspektor Gadget shipped gadgets in the binary. As of v0.31.0 these ***built-in*** Gadgets are still available and work as before but their use is discourage as they will be deprecate at some point. We encourage users to use ***image-based*** Gadgets going forward as they provide more features and decouple the eBPF programs from the Inspektor Gadget release process.
+
+### What is enrichment?
+
+The data that eBPF collects from the kernel includes no knowledge about Kubernetes, container
+runtimes or any other high-level user-space concepts. In order to relate this data to these high-level
+concepts and make the eBPF data immediately more understandable, Inspektor Gadget automatically
+uses kernel primitives such as mount namespaces, pids, etc. to infer which high-level
+concepts they relate to; Kubernetes pods, container names, DNS names, etc. The process of augmenting
+the eBPF data with these high-level concepts is called *enrichment*.
+
+Enrichment flows the other way, too. Inspektor Gadget enables users to do high-performance
+in-kernel filtering by only referencing high-level concepts such as Kubernetes pods, container
+names, etc.; automatically translating these to the corresponding low-level kernel resources.
+
+See the [enrichment documentation](docs/enrichment)
+
+### What is an operator
+
+In Inspektor Gadget, an operator is any part of the framework where an action is taken. Some operators are under-the-hood (i.e. fetching and loading Gadgets) while others are user-exposed (enrichment, filtering, export, etc.) and can be reordered and overridden.
 
-Inspektor Gadget is deployed to each node as a privileged DaemonSet.
-It uses in-kernel eBPF helper programs to monitor events mainly related to
-syscalls from userspace programs in a pod. The eBPF programs are run by
-the kernel and gather the log data. Inspektor Gadget's userspace
-utilities fetch the log data from ring buffers and display it. What eBPF
-programs are and how Inspektor Gadget uses them is briefly explained in
-the [architecture](docs/core-concepts/architecture.md) document.
+See the [operator documentation](docs/operators.md) for more information.
+
+### Further learning
+
+Use the [project documentation](/docs/_index.md) to learn more about:
+
+* [Overview](/docs/_index.md)
+* [Concepts](/docs/core_concepts/_index.md)
+* [Components](https://inspektor-gadget.io/docs/components)
+* [Gadgets](https://inspektor-gadget.io/docs/gadgets)
+* [Reference](https://inspektor-gadget.io/docs/reference)
+* [Contributing]()
```

Comment:
I suppose we need to update all of this to point to the website.

Reply:
Yes, please.